_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,11 +28,11 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any `iframe` element that has a negative number as a `tabindex` [attribute value][].
+This rule applies to any non-focusable `iframe` element that has focusable content.
```

Comment:
Are you changing to non-focusable because its more clear or is there a scenario that the negative tab-index doesn't catch? Asking because I do somewhat like how specific the negative tab-index is. 

Also, you will probably need to include a reference to [focusable](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/glossary/focusable.md) here. You should be able to add it by doing `[focusable][]`

Reply:
You will probably also need to change the description of the rule to mirror this.